src/reducers/versions.tsx

```diff
@@ -26,6 +26,7 @@ type VersionLicense = {
   url: string;
 };
 
+export type VersionEntryStatus = '' | 'M' | 'A' | 'D' | 'R' | 'C' | 'U';
```

Comment:
Can we really have `U`?

> U: file is unmerged (you must complete the merge before it can be committed)

Reply:
In `createInternalDiff`, we support `A, C, D, M, R` and we fallback to `M` otherwise. I don't think it is ultra relevant to know that here but now you know :p

Reply:
> Can we really have U?

theoretically, we don't have anything in place to avoid that status code to be issued but practically we don't. We don't do actual merges between versions to avoid complexity so `U` most certainly won't happen in real life.

Reply:
Ok, based on all that I will remove `U` from the list.